srcpkgs/dotnet-sdk/template

```diff
@@ -0,0 +1,75 @@
+# Template file for 'dotnet-sdk'
+pkgname=dotnet-sdk
```

Comment:
As per my other comment I think this should be `dotnet-sdk-bin`.

Reply:
That's a good point. Renamed to `dotnet-sdk-bin`